WireDomain/Tests/WireDomainTests/Synchronization/SyncManagerTests.swift

```diff
@@ -283,6 +316,44 @@ final class SyncManagerTests: XCTestCase {
         XCTAssertEqual(updateEventsRepository.stopReceivingLiveEvents_Invocations.count, 0)
     }
 
+    func testPerformSlowSync() async throws {
```

Comment:
```suggestion
    func test_performSlowSync_whenSuccessful() async throws {
```

Reply:
do we have a failure test like making sure we throw if one of the repository fails?

Reply:
I added one in https://github.com/wireapp/wire-ios/pull/1999/commits/fccfda883c009a5f7fb8378d54b31a6873b24b0e